lib/web/appserver.rb

```diff
@@ -268,6 +269,23 @@ def fill_general_all_no_in_database
     Database.instance.save_database if modified
   end
 
+  # サーバーの認証の設定
+  # とりあえずDigest認証のみ
+  def setup_server_authentication
+    setting = Inventory.load("global_setting", :global)
+    return unless setting["server-digest-auth.use"]
+
+    user = setting["server-digest-auth.user"]
+    hashed = !!setting["server-digest-auth.hashed-password"]
+    passwd = hashed ? setting["server-digest-auth.hashed-password"] : setting["server-digest-auth.password"]
+    
+    self.class.class_exec do
+      use Rack::Auth::Digest::MD5, {realm: 'narou.rb', opaque:'', passwords_hashed: hashed} do |username|
```

Comment:
__[rubocop]__
* Layout/SpaceInsideHashLiteralBraces: Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Layout/SpaceAfterColon: Space missing after colon. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Layout/SpaceInsideHashLiteralBraces: Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->